doc/connection-to-the-internet-of-things/deep-dive.md

```diff
@@ -1,13 +1,3 @@
 Should you wish to learn much more about this, please check: 
 
-   - [Github repository - IoT Agent UL 2.0](https://github.com/Fiware/iot.IoTagent-UL)
-   - [Play with the Docker container - IoT Agent UL 2.0](https://hub.docker.com/r/fiware/iotagent-ul/)
-   - [Browse the API](http://docs.telefonicaiotiotagents.apiary.io)
-   - [Read documentation - IoT Agent UL 2.0](http://fiware-iotagent-ul.readthedocs.io/en/latest/)
-   - [Latest stable release - IoT Agent UL 2.0](https://github.com/Fiware/iot.IoTagent-UL/releases/latest)
-   
-Other resources (advanced):
-   
-   - [IoT Agent JSON](https://github.com/Fiware/iot.IoTagent-JSON)
-   - [IoT Agent LWM2M](https://github.com/Fiware/iot.IoTagent-LWM2M)
-   - [IoT Agent Node.js Library - Framework to develop new IoT Agents](https://github.com/Fiware/iot.IoTagent-node-lib)
```

Comment:
Hi, @jmcanterafonseca, we talk about IoTAgent on introduction page and it redirect to github doc that have a section about Docker installation too